File: algorithms/python3/findNumbersWithEvenNumberOfDigits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        countDigits = 0
        countFinal = 0
        for i in range(len(nums)):
            countDigits = 0
            while (nums[i] != 0):
                nums[i] = nums[i] // 10
                countDigits += 1
            if (countDigits % 2 == 0):
                countFinal += 1
        return countFinal

File: algorithms/c/findNumbersWithEvenNumberOfDigits.c
int findNumbers(int* nums, int numsSize){
    int countDigits = 0;
    int countFinal = 0;
    for (int i = 0; i < numsSize; i++) {
        countDigits = 0;
        while (nums[i] != 0) {
            nums[i] = nums[i] / 10;
            countDigits++;
        }
        if (countDigits % 2 == 0) {
            countFinal++;
        }
    }
    return countFinal;
}
